Documentation/DSCResources/VMHostSSDCache/VMHostSSDCache.md

```diff
@@ -0,0 +1,53 @@
+# VMHostSSDCache
```

Comment:
Concider changing the name to VMHostCache

Reply:
Done.